docs/src/introduction/sway-toolchain.md

```diff
@@ -6,9 +6,15 @@ The Sway toolchain consists of several components.
 
 The "Fuel Orchestrator" [Forc](https://github.com/FuelLabs/sway/tree/master/forc) is our equivalent of Rust's [Cargo](https://doc.rust-lang.org/cargo/). It is the primary entry point for creating, building, testing, and deploying Sway projects.
 
-## Sway Language Server (`forc lsp`)
+## Sway Language Server (`forc-lsp`)
 
-The Sway Language Server `forc lsp` is provided to expose features to IDEs. Currently, only [Visual Studio Code is supported through a plugin](https://marketplace.visualstudio.com/items?itemName=FuelLabs.sway-vscode-plugin). Vim support is forthcoming, though [syntax highlighting is provided](https://github.com/FuelLabs/sway.vim).
+The Sway Language Server `forc-lsp` is provided to expose features to IDEs, which you can install with cargo:
+
+```console
```

Comment:
`console` if you're going to do `$ command_name` followed by output, `sh` if it's just the commands.

````suggestion
```sh
````